internal/cli/refresher_opts.go

```diff
@@ -25,6 +25,8 @@ import (
 	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
+var TokenRefreshed bool
```

Comment:
[nit] let's make it readonly just in case  
```suggestion
var tokenRefreshed bool
func TokenRefreshed() bool {
  return tokenRefreshed
}
```

Reply:
Unfortunately I can't because I need to set it on tests, across different packages